resources/views/header-script.blade.php

```diff
@@ -16,7 +16,8 @@
             window.dataLayer.push({!! $item->toJson() !!});
         @endforeach
 
-        window.addEventListener('push_to_data_layer', (event) => window.dataLayer.push(event.detail))
+        window.addEventListener('push_to_data_layer', (event) => window.ElevarPushToDataLayer(event.detail))
```

Comment:
@BaffourAdu were you able to address the changes we discussed?

```suggestion
		@if(settings(\Astrogoat\Elevar\Settings\ElevarSettings::class)->isEnabled())
	        window.addEventListener('push_to_data_layer', (event) => window.ElevarPushToDataLayer(event.detail))
        @else
            window.addEventListener('push_to_data_layer', (event) => window.dataLayer.push(event.detail))
        @endif
```

Disclaimer: I have not tested this code.